x-pack/plugins/security_solution_serverless/public/get_started/step_links/alerts_link.tsx

```diff
@@ -21,21 +21,13 @@ const AlertsLinkComponent = () => {
     onClick,
   });
   return (
-    <FormattedMessage
-      id="xpack.securitySolutionServerless.getStarted.togglePanel.explore.step1.description2.linkText"
-      defaultMessage="Visit the {link} now to confirm your organization is secure!"
-      values={{
-        link: (
-          <LinkAnchor onClick={onLinkClicked} id={SecurityPageName.alerts}>
-            <FormattedMessage
-              id="xpack.securitySolutionServerless.getStarted.togglePanel.explore.step1.description2.link"
-              defaultMessage="alerts page"
-            />
-          </LinkAnchor>
-        ),
-      }}
-    />
+    <LinkButton onClick={onLinkClicked} id={SecurityPageName.alerts} fill>
```

Comment:
Same, `onLinkClicked` is not needed
```suggestion
    <LinkButton onClick={onClick} id={SecurityPageName.alerts} fill>
```